nodes:
- area_name: LOGIMAT_C1
  area_type: corridor
  id: 1
  x: -1.34
  y: 0.56
- area_name: LOGIMAT_C2
  area_type: corridor
  id: 2
  x: -1.77
  y: 2.41
- area_name: BRSU_area
  area_type: area
  id: 3
  x: -3.16
  y: 4.31
- area_name: BRSU_area
  area_type: area
  id: 4
  x: -3.62
  y: 1.79
- area_name: BRSU_area
  area_type: area
  id: 5
  x: -4.85
  y: 2.03
- area_name: BRSU_area
  area_type: area
  id: 6
  x: -3.28
  y: -0.07
- area_name: BRSU_area
  area_type: area
  id: 7
  x: -5.26
  y: -0.23
connections:
- - 1
  - 2
- - 2
  - 3
- - 4
  - 3
- - 4
  - 2
- - 5
  - 3
- - 5
  - 7
- - 6
  - 7
